src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx

```diff
@@ -54,12 +55,22 @@ const BlastInputSequencesHeader = (props: Props) => {
 
   const appendEmptyInput = () => {
     dispatch(updateEmptyInputDisplay(true));
+
+    // give React time to add the input box
+    setTimeout(() => scrollToLastInputBox(), 0);
   };
 
   const onSwitchToSpecies = () => {
     dispatch(switchToSpeciesStep());
   };
 
+  const scrollToLastInputBox = () => {
+    const sequenceTextarea = document
+      .querySelector(`.${sequenceBoxStyles.inputSequenceBox}:last-child`)
+      ?.querySelector('textarea');
```

Comment:
No reason not to include the `textarea` in the css of the previous querySelector.

Reply:
I remembered trying to put it together in the same css but there was an error which i couldn't understand. And i have just tried exactly your code and this is the error that i am seeing:


![image](https://user-images.githubusercontent.com/6040010/155522451-c08d53c0-bf3d-48f7-b64f-6a27e7b60f70.png)


And strange enough when i do it separately it works.

Reply:
@azangru i have made the suggested changes and its now ready for another review. Thanks